api/organisations/models.py

```diff
@@ -148,8 +158,12 @@ def update_plan(self, plan_id):
         self.max_api_calls = get_max_api_calls_for_plan(plan_metadata)
         self.save()
 
-    @hook(AFTER_CREATE)
+    @property
+    def can_auto_upgrade_seats(self) -> bool:
+        return self.plan in settings.AUTO_SEAT_UPGRADE_PLANS
+
     @hook(AFTER_SAVE, when="cancellation_date", has_changed=True)
+    @hook(AFTER_SAVE, when="subscription_id", has_changed=True)
```

Comment:
What's the logic here?

Reply:
It is because we automatically create a subscription with every org creation, and a subscription_id change will indicate a move to a paid plan

Reply:
Ah, ok, yes, I didn't follow it through far enough to see that we send a flag of `is_paid` to Mailerlite. This is a good catch in that case, nice work! 😄